code/parsers/curl.py

```diff
@@ -39,14 +39,15 @@ def get_document_dependencies(self, document: str) -> List[ExtractedDependency]:
         dependencies = []
         curl_dependencies = self.curl_extractor_regex.findall(document)
         for match in curl_dependencies:
-            url_extract = self.url_extractor_regex.findall(match[3])[0]
+            curl_extraction_source = match[0]
+            url_extract = self.url_extractor_regex.findall(curl_extraction_source)[0].strip()
```

Comment:
During testing, I was able to get this parser to fail here with:

  File "/workspaces/build-inspector/code/./parsers/curl.py", line 43, in get_document_dependencies
    url_extract = self.url_extractor_regex.findall(curl_extraction_source)[0].strip()
IndexError: list index out of range

The specific matching text for this failure was:
"curl --cacert /tmp/build/d502a588/root.crt'\\\\r\r"

Please provide handling for this specific failure, or avoid this match